contrib/for-tests/netexec/netexec.go

```diff
@@ -191,11 +193,53 @@ func dialUDP(request string, remoteAddress *net.UDPAddr) (string, error) {
 
 func shellHandler(w http.ResponseWriter, r *http.Request) {
 	log.Println(r.FormValue("shellCommand"))
-	output, err := exec.Command(shellPath, "-c", r.FormValue("shellCommand")).CombinedOutput()
-	assertNoError(err)
```

Comment:
To control the output of the logging. It's not a must have by any means but I found it a bit easier to read in the logs.